pages/BasicCalculator.js

```diff
@@ -0,0 +1,9 @@
+exports.BasicCalculator = class BasicCalculator {
+    constructor(page) {
+        this.page = page;
+    }
+
+    async goto() {
+        await this.page.goto('https://testsheepnz.github.io/BasicCalculator');
```

Comment:
šis failas yra nereikalingas, kadangi jo panaudojimas kaip matau calculator-homework.spec.js faile užkomentuotas.